documents/specs/srs.md

```diff
@@ -0,0 +1,196 @@
+# Software Requirements Specification
+
+* [Software Requirements Specification](#software-requirements-specification)
+  * [Use Cases](#use-cases)
+  * [Overall Description](#overall-description)
```

Comment:
Can we move the Overall Description items into separate files in the future? to make it easier to view and edit files.
for example next file will be projectManagement-srs.md. Or will it on the contrary complicate the work?